src/renderer/components/ui/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import classNames from 'classnames'
+
+type ButtonProps = {
+  children: any
+  onClick?: any
+  className?: any
+  style?: Record<string, string | number>
```

Comment:
Offering `className` and `style` is a backdoor for hacky CSS patterns, I'd try to _not_ allow these props except of when we can't get there yet as changing that now would lead to a large refactoring.

If you see the possibility to not offer these properties I'd strongly recommend it.

I suggest to rather introduce very clear properties to allow changing the "variant" of the component (as you do for example already with `type`, that is a nice pattern). If the component gets too powerful and flexible its rather a sign to introduce a new component.